wlib/strings/StaticString.h

```diff
@@ -22,53 +30,63 @@ class StaticString {
 	 *
 	 * @param str char string
 	 */
-	StaticString<tSize>(const char* str){
-		uint16_t destSize = (uint16_t)ceil(fmin((uint16_t) strlen(str), capacity()));
-		strncpy(m_buffer, str, destSize);
-		m_buffer[destSize] = '\0';
-		len = destSize;
+	explicit StaticString<tSize>(const char* str){
+		m_len = (uint16_t) strlen(str);
+		if (m_len > capacity()) m_len = capacity();
+
+		strncpy(m_buffer, str, m_len);
+		m_buffer[m_len] = '\0';
 	}
 
 	/**
 	 * Constructor creates string using static string object
 	 *
 	 * @param str @code StaticString object
 	 */
-	StaticString<tSize>(const StaticString<tSize> &str){
-		StaticString(str.c_str());
-	}
+	StaticString<tSize>(const StaticString<tSize> &str) : StaticString{str.c_str()}{}

```

Comment:
We need a constructor for non `const`

Reply:
Also make `const` reference as `const StaticString<tSize> const& str` whenever the reference is not modified